src/components/DeploymentQuickRun.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <p-overflow-menu-item class="run-menu__overflow-menu-item" @click="run">
```

Comment:
```suggestion
  <p-overflow-menu-item @click="run">
```
This class isn't needed (and was incorrect)